detection_rules/packaging.py

```diff
@@ -112,7 +112,18 @@ def manage_versions(rules: List[TOMLRule], deprecated_rules: list = None, curren
 
         if save_changes:
             if changed_rules or (new_rules and add_new):
-                current_versions.update(new_rules if add_new else {})
+                if add_new:
+                    for rule in new_rules:
+                        current_versions[rule.id] = {"rule_name": rule.name,
```

Comment:
nit: the dict contents should start on a new, indented line

Reply:
alternate idea: build the dict earlier where it's not indentend so far and save 5 lines

Reply:
added a method to return the dict e60fab52bb263433b9a9f0fb4edc3c7766b4626c